catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -528,9 +536,10 @@ public SourceInfoResponse getFanoutSourceInfo(SourceInfoRequest sourceInfoReques
             if (ids != null && !ids.isEmpty()) {
                 for (String id : ids) {
                     if (!id.equals(this.getId())) {
-                        SourceUnavailableException sourceUnavailableException = new SourceUnavailableException(
-                                "Unknown source: " + id);
-                        LOGGER.warn("Throwing SourceUnavilableExcption for unknown source: {}", id,
+                        SourceUnavailableException sourceUnavailableException =
+                                new SourceUnavailableException("Unknown source: " + id);
+                        LOGGER.warn("Throwing SourceUnavilableExcption for unknown source: {}",
```

Comment:
I know it's not your change, but could we please fix the multiple typos in this message.